### PR TITLE
remove unnecessary collect

### PR DESCRIPTION
**Pull request by jiacai2050**

### CODE REVIEW

File: src/xor8/builder.rs [hunk 1]
```diff
@@ -123,8 +128,7 @@ where H: BuildHasher + Clone
     /// Build bitmap for keys that where previously inserted using [Xor8Builder::insert],
     /// [Xor8Builder::populate] and [Xor8Builder::populate_digests] method.
     pub fn build(&mut self) -> Result<Xor8<H>, crate::Error> {
-        let digests = self.digests.iter().copied().collect::<Vec<u64>>();
```
Comment (jiacai2050) [inline src/xor8/builder.rs:151]: Mainly remove this `collect`

File: src/xor8/builder.rs [hunk 2]
```diff
@@ -160,14 +172,26 @@ where H: BuildHasher + Clone
...
+            } else {
+                for key in &self.digests {
+                    let hs = ff.get_h0h1h2(*key);
+                    sets0[hs.h0 as usize].xor_mask ^= hs.h;
+                    sets0[hs.h0 as usize].count += 1;
+                    sets1[hs.h1 as usize].xor_mask ^= hs.h;
+                    sets1[hs.h1 as usize].count += 1;
+                    sets2[hs.h2 as usize].xor_mask ^= hs.h;
+                    sets2[hs.h2 as usize].count += 1;
+                }
```
Comment (Rachelint) [inline src/xor8/builder.rs]: Can remove duplicated codes here?

**Comment by Rachelint** [comment]
LGTM

**Review by Rachelint — APPROVED** [review]
LGTM

**Pull request merged**